.bumpversion.cfg

```diff
@@ -0,0 +1,20 @@
+[bumpversion]
+current_version = 3.4.1
```

Comment:
I did some research, and galaxy allows versions like "3.4.1-1" and "3.4.1-dev1" for collections. [example 1](https://galaxy.ansible.com/ansible/netcommon) [example 2](https://galaxy.ansible.com/freeipa/ansible_freeipa)

So we need to do 2 different variables: 1 for the version of pulp installed (3.4.1), and 1 for the version of pulp_installer (which could be 3.4.1 or either of those examples.)

Reply:
this `current_version` is what `bumpversion` looks for to replace

Reply:
Right, but it is not sufficient for pulp_installer. We care about 2 versions: The current version of Pulp, and the current version of pulp_installer. The 2 versions can be identical, or the Pulp version can be a substring of pulp_installer's version.

Reply:
[bump2version](https://github.com/c4urself/bump2version) seems to support only one version, I think for pulpcore version we probably need to add another arg and handle it inside the python script.
Like:
```
python .github/release.py patch --pulpcore 3.5.0
```

Reply:
@fao89 I agree.

Reply:
Right now index.md has 15 occurrences of "3.4.1"
It is hard to treat in a script, how can the script knows which one is pulpcore or pulp_installer version?

Reply:
@fao89 Well, I figured that would be the responsibility of the dev calling the script.

But keep in mind that the pulpcore version will always be a substring of the pulp_installer version. So we can replace all instances of the pulp_installer version, then replace all instances of the pulpcore version. And if they're the same, great, no changes to make when we try to replace the pulpcore version.

Do you agree with my assessment?

Reply:
yep, I was trying to think, the better I got is:
always append pulpcore when referring to it: pulpcore-3.4.1
so the script would treat 3.4.1 as installer reference and pulpcore-3.4.1 as pulpcore

Reply:
In python the strategy is to deduplicate all of the references to one place and have bump2version (or similar) bump that one place. In reading about pulp_installer and pulpcore having usually the same but sometimes not versions, we should probably de-duplicate to two things somehow. I think this is what you were saying more or less @mikedep333 .

I think we want to fully automate things so generally we need to find another way than having a dev manually do it.